src/components/FormComponents/AssetPicker.js

```diff
@@ -94,21 +94,51 @@ function codeValidator(value, ignoreType = false) {
   }
 }
 
-const stringFormRadioOptions = { native: "Native", issued: "Issued" };
-function AssetStringPicker({ value, onUpdate, ...props }) {
-  const isNative = !value.includes(":");
+const ASSET_TYPES = {
+  none: "none",
+  native: "native",
+  issued: "issued",
+};
+const stringFormRadioOptions = {
+  [ASSET_TYPES.native]: "Native",
+  [ASSET_TYPES.issued]: "Issued",
+};
```

Comment:
Nit: is this just so the radio labels are capitalized? is there any value in just using ASSET_TYPES.native and using a css text transform to capitalize instead of creating a new obj?

Reply:
Good point. In this case the label happens to be the capitalized form of the word we're using internally, but I don't think I want to couple it so that it _has_ to be that.